plugin/PDFView.qml

```diff
@@ -223,8 +311,7 @@ SilicaFlickable {
                 selection: pdfSelection
 
                 onLinkClicked: base.linkClicked(linkTarget, contextHook)
-                onGotoClicked: base.goToPage(page - 1, top, left,
-                                             Theme.paddingLarge, Theme.paddingLarge)
+                onGotoClicked: base.gotoClicked(page, top, left)
```

Comment:
That's right. The toolbar was made visible on gotoLink signal before, but then, I switched to listen to change of canMoveBack which more logical IMHO. Forgot to remove the signal then. Corrected.